src/spac/visualization.py

```diff
@@ -377,44 +377,28 @@ def hierarchical_heatmap(
         Name of the annotation in adata.obs to group by and calculate mean
         intensity.
     features : list or None, optional
-        List of feature names (e.g., genes) to be included in the
+        List of feature names (e.g., markers) to be included in the
         visualization. If None, all features are used. Default is None.
     layer : str, optional
         The name of the `adata` layer to use to calculate the mean intensity.
-        Default is None.
-    row_cluster : bool, optional
-        If True (default), perform hierarchical clustering on the rows.
-        This corresponds to clustering the features if `swap_axes` is False,
-        or the annotations if `swap_axes` is True.
-    col_cluster : bool, optional
-        If True (default), perform hierarchical clustering on the columns.
-        This corresponds to clustering the annotations if `swap_axes` is False,
-        or the features if `swap_axes` is True.
+        If not provided, uses the main matrix. Default is None.
+    cluster_feature : bool, optional
+        If True, perform hierarchical clustering on the feature axis.
+        Default is False.
+    cluster_annotations : bool, optional
+        If True, perform hierarchical clustering on the annotations axis.
+        Default is False.
     standard_scale : int or None, optional
         Whether to standard scale data (0: row-wise or 1: column-wise).
         Default is None.
-    z_score : int or None, optional
-        Specifies the axis for z-score normalization.
-        - z_score=1: Column-wise standardization. In the default orientation,
-          this applies z-score normalization across features, aiding in
-          comparing different annotations' expression intensities for a
-          specific feature.
-        - z_score=0: Row-wise standardization. In the default orientation,
-          this applies normalization across annotations, helping compare
-          various features within a single annotation.
-        When `swap_axes` is True, the roles of rows and columns are reversed.
-        Default is None.
+    z_score : str, optional
+        Specifies the axis for z-score normalization. Can be "feature" or
+        "annotation". Default is "annotation".
     swap_axes : bool, optional
-        Switches the axes of the heatmap, effectively transposing the dataset.
-        Default orientation: Rows represent annotations and columns represent
-        features. When True: Rows represent features and columns represent
-        annotations. This also switches the behavior of parameters like
-        z_score, row_cluster, and col_cluster. Default is False.
+        If True, switches the axes of the heatmap, effectively transposing

```

Comment:
@fangliu117, please add the behavior of the default: what is in horizontal, and what is in vertical?

Reply:
Yes, I have revised the docstring.